index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Ruben's Blog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/layout.css">
+
+    <script id="blog-template" type="text/x-handlebars-template">
+      <article class="{{class}}" data-story="{{story}}" data-class={{class}}>
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+            {{publishedOn}}
+          </div>
+        </header>
+        <section class="blog-body">{{{body}}}</section>
+        <a href="#" class="read-on">Read on &rarr;</a>
+      </article>
+    </script>
+  </head>
+  <body>
+    <header class="site-header">
+      <section>
+        <h1>Welcome to my Sports Blog!</h1>
+      </section>
+      <nav class="main-nav">
+        <div class="icon-menu"><img src="img/bball.png"></div>
+          <ul>
+            <li><a href=""> Home</a></li>
```

Comment:
You need to have a class of tab on either the li or the a to make the tab navigation work.